.travis.yml

```diff
@@ -1,19 +1,23 @@
-sudo: false
+os:
+  - linux
+  - osx
```

Comment:
We don't need to test this on macos. Historically, the queue has been really long and this has no os specific code.

Reply:
My bad; I copied it from another repo without thinking too much about it.

Reply:
(which repo? We have a bunch of outdate CI scripts...)

Reply:
I think I got it from the `go-maddr-filter` repo: https://github.com/libp2p/go-maddr-filter/blob/master/.travis.yml.

I can remove the osx build from there as well (and remove it from other repos that don't have OS specific code as I encounter such instances).

Reply:
If you're willing to spend the time at it, that would be great! We usually just wait for it to sufficiently annoy someone.